modules/sandboxed-containers-create-kata-config-resource-cli.adoc

```diff
@@ -0,0 +1,79 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-create-kataconfig-rsource-cli_{context}"]
+= Creating the KataConfig custom resource using the CLI
+
+You must create one `KataConfig` custom resource (CR) to install the Kata runtime on your nodes. When you create the `KataConfig` CR, it triggers the {sandboxed-containers-operator} to do the following:
+
+* Install the needed {op-system} extensions, such as QEMU and `kata-containers`, on your {op-system} node.
+* Ensure that the link:https://github.com/cri-o/cri-o[CRI-O] runtime is configured with the correct Kata runtime handlers.
+* Create a `RuntimeClass` custom resource with the necessary configurations for additional overhead.
+
+[NOTE]
+====
+The Kata runtime is installed on all worker nodes by default. To install the Kata runtime only on selected nodes, you must label the nodes first, and then create the `KataConfig` CR and include the label in the YAML file.
+====
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the {sandboxed-containers-operator}.
+
+.Procedure
+
+. Create a YAML file with the following manifest:
++
+[source,yaml]
+----
+apiVersion: kataconfiguration.openshift.io/v1
+kind: KataConfig
+metadata:
+  name: cluster-kataconfig
+----
+
+. (Optional) If you want to install the Kata runtime only on selected nodes, create a YAML file that includes the label in the manifest:
++
+[source,yaml]
+----
+apiVersion: kataconfiguration.openshift.io/v1
+kind: KataConfig
+metadata:
+  name: cluster-kataconfig
+spec:
+  kataConfigPoolSelector:
+    matchLabels:
+    <label_key>: '<label_value>'
+----
+
+. Create the `KataConfig` resource:
++
+[source,terminal]
+----
+$ oc create -f <file name>.yaml
+----
+
+The new KataConfig CR is created and begins to install the kata runtime on the worker nodes.
```

Comment:
+1 to `KataConfig`. Plus "kata runtime" is not capitalized here.

Reply:
Fixed.